Dockerfile

```diff
@@ -20,15 +20,6 @@ COPY controllers/ controllers/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
 
-# Add labels to image
```

Comment:
Downstream products across IBM and RH now use the downstream packaging methods, this is hence not required as yet.